scripts/ask_qdrant.py

```
@@ -0,0 +1,51 @@
+import argparse
+import json
+import os
+import sys
+from pathlib import Path
+import datetime as _dt
+
+
+def ensure_env(var: str) -> str:
+    value = os.getenv(var)
+    if not value:
+        raise SystemExit(f"Missing required environment variable: {var}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)